Authentication Methods Policies API doc updates (#9924)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/fido2keyrestrictions.md

---
title: "fido2KeyRestrictions resource type"

> description: "A representation of key restrictions that are enforced as part of FIDO2 security keys authentication methods policy"

Reviewer: ```suggestion
description: "Represents the key restrictions that are enforced as part of the FIDO2 security keys authentication methods policy."
```

author: "mmcla"
localization_priority: Normal
ms.prod: "microsoft-identity-platform"
doc_type: "apiPageType"
---

# fido2KeyRestrictions resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> A representation of key restrictions that are enforced as part of [FIDO2 security keys authentication methods policy](../resources/fido2authenticationmethodconfiguration.md).

Reviewer: Represents the key restrictions that are enforced as part of the [FIDO2 security keys authentication methods policy](../resources/fido2authenticationmethodconfiguration.md).

Author: Fixed

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |aaGuids|String collection|A collection of Authenticator Attestation GUIDs. AADGUIDs define key types and manufacturers.|

Reviewer: ```suggestion
|aaGuids|String collection|A collection of Authenticator Attestation GUIDs (AADGUID). AADGUIDs define key types and manufacturers.|
```

Is there a link to any conceptual documentation about AADGUIDs?

Author: No
